src/widgets/DataTable/DataTableRow.js

```diff
@@ -324,18 +319,19 @@ const DataTableRow = React.memo(
             }
 
             case COLUMN_TYPES.DROP_DOWN: {
-              const { isVaccine = false } = rowData ?? {};
+              const { isVaccine = false, hasVariance } = rowData ?? {};
 
               const disabledConditions = {
                 [COLUMN_KEYS.CURRENT_VVM_STATUS]: !isVaccine,
+                [COLUMN_KEYS.REASON_TITLE]: !hasVariance,
               };
 
-              const disabledVVMStatus = disabledConditions[columnKey];
+              const extraDisabledCondition = disabledConditions[columnKey];
```

Comment:
![blewmind](https://user-images.githubusercontent.com/43223637/99920997-49b1ea00-2d8c-11eb-93b9-51c4d5985a39.gif)

...maybe something for a tidy up later?